src/libcalamares/Settings.h

```diff
@@ -176,6 +178,7 @@ class DLLEXPORT Settings : public QObject
     bool m_promptInstall;
     bool m_disableCancel;
     bool m_disableCancelDuringExec;
+    bool m_hideBackAndNextDuringExec;
```

Comment:
` = false;` or it needs to be initialized in the constructors in `Settings.cpp`. Come to think of it, each of these bools could be `=` assigned here in the class declaration, and the initializer lists in the constructor simplified.

Reply:
Okay. I've looked into the default `settings.conf` file and tried to extract values from there. Kindly just **verify** them once.